content/docs/06.workflow-components/15.descriptions.md

```diff
@@ -10,11 +10,50 @@ The `description` property is a string that supports [Markdown](https://en.wikip
 You can add a `description` property on:
 - Flows
 - Inputs
-- Outputs
```

Comment:
This looks to be a miss. How can Outputs have description?! Hence, removed this.

Reply:
Believe this is for the recent change in 0.15.0 which allowed outputs to be strongly typed. Documentation mentions that description can be added but I haven't tried it yet.

https://kestra.io/docs/workflow-components/outputs#pass-data-between-flows-using-flow-outputs